suite-native/icons/src/Icon.tsx

```diff
@@ -0,0 +1,127 @@
+import { useMemo } from 'react';
+import { Platform, Text as RNText } from 'react-native';
+import Animated, { SharedValue, useAnimatedStyle } from 'react-native-reanimated';
+
+// @ts-expect-error This is not public RN API but it will make Text noticeable faster https://twitter.com/FernandoTheRojo/status/1707769877493121420
+import { NativeText } from 'react-native/Libraries/Text/TextNativeComponent';
+
+import codepoints from '@suite-common/icons/iconFontsMobile/TrezorSuiteIcons.json';
+import { useNativeStyles } from '@trezor/styles';
+import { Color, Colors, CSSColor } from '@trezor/theme';
+import { MOBILE_ICON_FONT_NAME } from '@suite-common/icons';
+
+export type IconColor = Color | CSSColor;
+export type AnimatedIconColor = Color | CSSColor | SharedValue<CSSColor>;
+
+export const icons = codepoints;
+
+/**
+ * @description If you need to add a new icon, please follow these steps:
+ * 1. Add the icon name to the file `generateIconFonts.ts`.
+ * 2. Run `yarn generate-icons` to generate the new icon font.
+ */
+export type IconName = keyof typeof codepoints;
+
+export const iconSizes = {
+    extraSmall: 8,
+    small: 12,
+    medium: 16,
+    mediumLarge: 20,
+    large: 24,
+    extraLarge: 32,
+} as const;
+
+export type IconSize = keyof typeof iconSizes;
+
+export const getIconSize = (size: IconSize | number) =>
+    typeof size === 'string' ? iconSizes[size] : size;
+
+// NativeText improves the performance of the text rendering, but unfortunately it does not support iOS Accessibility font enlarging.
+// Since iOS devices have enough computational power and the text optimization is not crucial, the NativeText is used only for Android.
+const DefaultTextComponent: typeof RNText = Platform.select({
+    android: NativeText,
+    ios: RNText,
+});
+
+export type IconProps = {
+    name: IconName;
+    size?: IconSize | number;
+    color?: IconColor;
+};
+
+export const Icon = ({ name, size = 'large', color = 'iconDefault' }: IconProps) => {
```

Comment:
Not sure what exact change caused Device info and Open passphrase buttons to be cut off, but looks ok on develop.

<img width="374" alt="image" src="https://github.com/user-attachments/assets/5427dbaa-bf9f-4344-9ad3-d0d36b38405c">

Reply:
I am not able to reproduce this one, can you please try again? Maybe it was some random temp issue.